main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"./routes"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+func Router() *mux.Router {
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc("/", indexRoute)
+	router.HandleFunc("/bitcoin", routes.GetBicoins).Methods("GET")
+	router.HandleFunc("/bitcoin/{id}", routes.GetBicoin).Methods("GET")
+	return router
+}
```

Comment:
I feel like this router could be created only once then destroyed after finishing the tests.